javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,100 @@
+const recipeObject = {
+  id: 1,
+  title: "Gløgg",
+  picture_url: "https://upload.wikimedia.org/wikipedia/commons/thumb/e/e2/Gl%C3%B6gg_kastrull.JPG/800px-Gl%C3%B6gg_kastrull.JPG",
+  ingredients: [
+      { NAME: "Orange zest", AMOUNT: "0.5" },
```

Comment:
upper case!
The object properties (NAME, AMOUNT) are in uppercase, camelCase would be better